src/util.rs

```diff
@@ -58,11 +58,21 @@ fn test_parse_size() {
 }
 
 pub fn read_passphrase() -> io::Result<String> {
+    // The environment variable should only be used for testing
+    if let Ok(pass) = env::var("RDEDUP_PASSPHRASE") {
+        printerr!("Using passphrase set in RDEDUP_PASSPHRASE\n");
+        return Ok(pass);
+    }
     printerr!("Enter passphrase to unlock: ");
     rpassword::read_password()
 }
 
 pub fn read_new_passphrase() -> io::Result<String> {
+    // The environment variable should only be used for testing
+    if let Ok(pass) = env::var("RDEDUP_PASSPHRASE") {
```

Comment:
Let's keep it right now like that since it unblocks e2e testing, but we should remove this at some point. It's not a good practice security-wise. Generally, the best way to get a password from the user in a non-interactive way is reading it from a file. Having it in environment exposes it to too many programs, and unaware users will just set it with `export BLA=foo` command which is globally visible in `ps -All` etc. so also potentially easily stolen.